bin/hdfs-mesos-datanode

```diff
@@ -2,24 +2,5 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-if [ -z "$JAVA_HOME" ]; then
-  JAVA_HOME_DIR=$(dirname $(readlink -f $(which java)))
-  if [ -f $JAVA_HOME_DIR/../../bin/java ]; then
-    export JAVA_HOME=$JAVA_HOME_DIR/../..
-  elif [ -f $JAVA_HOME_DIR/../bin/java ]; then
-    export JAVA_HOME=$JAVA_HOME_DIR/..
-  else 
-    echo “Error: Could not determine JAVA_HOME”
-    exit 1;
-  fi
-fi
-
-trap "{ $DIR/mesos-killtree "$$" ; exit 0; }" EXIT
-
-function run_datanode() {
-  while [ true ] ; do
-    $DIR/hdfs datanode
-  done
-}
-
-run_datanode
+$DIR/hdfs-mesos-env
```

Comment:
I'd prefer if we kept the `while [ true ] do ; ... ; done` stuff for now.

Think of it as cheap and easy HA.